src/components/ScanQRModal/index.tsx

```diff
@@ -88,16 +91,17 @@ export const ScanQRModal = ({ isOpen, onClose, onScan }: Props): React.ReactElem
         ) : (
           <QrReader
             legacyMode={!useWebcam}
-            onError={(err) => onFileScannedResolve(err, null)}
-            onScan={(data) => onFileScannedResolve(null, data)}
+            onError={(err: Error) => onFileScannedResolve(err, null)}
```

Comment:
can we just pass the function here? `onError={onFileScannedResolve}`

Reply:
TypeScript will complain because the callback needs 2 arguments and `onError` only passes 1.